packages/typescript/src/components/MemberIdentifier.tsx

```diff
@@ -0,0 +1,66 @@
+import {
+  Children,
+  isRefkey,
+  OutputSymbol,
+  Refkey,
+  useBinder,
+} from "@alloy-js/core";
+
+/**
+ * Common props shared by both variants
+ */
+export interface CommonMemberExpressionProps {
```

Comment:
I don't like how MemberIdentifier is concerned with both rendering the name of the symbol it is provided and exposing a `nullish` prop so that `MemberChainExpression` can pick it up. This feels like a conflation of concerns, and is a bit confusing (how can a member identifier be nullish??).

Maybe better would be to have a component like `MemberExpressionPart` which has `nullish` on it. As a baseline experience, when you need a ?., you have to wrap in that component (where today you just pass nullish).

But I still have a vague worry that something is missing. This does not work for optional calls for example. Hmm. Need to ponder.